test/elixir_internal_certificate_web/controllers/api/v1/user_search_controller_test.exs

```diff
@@ -0,0 +1,155 @@
+defmodule ElixirInternalCertificateWeb.Api.V1.UserSearchControllerTest do
+  use ElixirInternalCertificateWeb.ConnCase, async: true
+
+  describe "GET index/2" do
+    test "when logged in user, returns list of keywords and 200 status", %{conn: conn} do
+      user = insert(:user)
+      insert(:user_search, keyword: "keyword 1", status: :success, user: user)
+      insert(:user_search, keyword: "keyword 2", status: :pending, user: user)
+      _another_user_keyword = insert(:user_search)
+
+      conn =
+        conn
+        |> token_auth_user(user)
+        |> get(Routes.api_user_search_path(conn, :index))
+
+      assert %{
+               "data" => [
+                 %{
+                   "attributes" => %{
+                     "keyword" => "keyword 1",
+                     "status" => "success"
+                   },
+                   "id" => _,
+                   "relationships" => %{},
+                   "type" => "user_search"
+                 },
+                 %{
+                   "attributes" => %{
+                     "keyword" => "keyword 2",
+                     "status" => "pending"
+                   },
+                   "id" => _,
+                   "relationships" => %{},
+                   "type" => "user_search"
+                 }
+               ],
+               "included" => []
```

Comment:
should we test for the `meta` part as well

Reply:
Updated in [3106803](https://github.com/nvminhtue/elixir_internal_certificate/pull/44/commits/3106803ed2ee151f54bb53ae2623eff0fe12990b)